dali/test/python/test_operator_numba_func_gpu.py

```diff
@@ -83,11 +89,13 @@ def test_numba_func():
         yield _testimpl_numba_func, shape, dtype, run_fn, out_types, in_types, outs_ndim, ins_ndim, blocks, threads_per_block, setup_fn, batch_processing, expected_out
 
 
+@with_setup(check_env_compatibility)
 @raises(AssertionError, glob='Currently batch processing for GPU is not supported.')
 def test_batch_processing_assertion():
     _testimpl_numba_func([(10, 5)], np.uint8, set_all_values_to_255_sample, [dali_types.UINT8], [dali_types.UINT8], [2], [2], [1, 1, 1], [10, 5, 1], None, True, [np.full((10, 5), 255, dtype=np.uint8)])
 
 
+@with_setup(check_env_compatibility)
 @raises(RuntimeError, regex="Shape of input [0-9]+, sample at index [0-9]+ doesn't match the shape of first sample")
```

Comment:
Shouldn't all numba test be tagged with `@attr(numba)`? Or we run this particular file only in specific job? I assume it is picked up by the test_operator_* job?

Reply:
They were not tagged in CPU variant, so they are not tagged for GPU as well. 

Actually, the only place where this tag is used is in the functional API test.

Reply:
I am a bit surprised, as we have tests that disallow numba, but maybe we just assume it is fine to test it in the operators_* job.